code/game/objects/items/storage/secure.dm

```diff
@@ -167,7 +167,6 @@
 	icon_locking = "safeb"
 	icon_sparking = "safespark"
 	desc = "Excellent for securing things away from grubby hands."
-	force = 8
```

Comment:
Is this intended?

Reply:
People kept saying that the secure safe showing up as a weapon was a bug. I don't see how it could be used as a weapon so I didnt think it needed a force value, but I can revert it if need be

Reply:
Alright.